projects/vdk-plugins/vdk-trino/tests/test_vdk_templates.py

```diff
@@ -27,42 +28,54 @@
 def trino_move_data_to_table_break_tmp_to_target(
     obj, from_db: str, from_table_name: str, to_db: str, to_table_name: str
 ):
-    if from_table_name == "tmp_dw_scmdb_people" and to_table_name == "dw_scmdb_people":
+    if from_table_name.startswith("tmp_dw_people") and to_table_name.startswith(
+        "dw_people"
+    ):
         obj.drop_table(from_db, from_table_name)
     return org_move_data_to_table(obj, from_db, from_table_name, to_db, to_table_name)
 
 
 def trino_move_data_to_table_break_tmp_to_target_and_restore(
     obj, from_db: str, from_table_name: str, to_db: str, to_table_name: str
 ):
-    if from_table_name == "tmp_dw_scmdb_people" and to_table_name == "dw_scmdb_people":
+    if from_table_name.startswith("tmp_dw_people") and to_table_name.startswith(
+        "dw_people"
+    ):
         obj.drop_table(from_db, from_table_name)
-    if (
-        from_table_name == "backup_dw_scmdb_people"
-        and to_table_name == "dw_scmdb_people"
+
+    if from_table_name.startswith("backup_dw_people") and to_table_name.startswith(
```

Comment:
Why moving tests from an exact table name to a non-exact one?
I now see that some of the tests source & target tables are changed to dynamic ones (and some are kept static), why?

Reply:
The schema name is dynamic and different for each test case because some tests were failing because they were written in the same table which was one of the reasons for the intermittent failures of the tests (explained it also in the commit description). 

I renamed some tables to make it easier to troubleshoot - having scmdb_people everywhere in the logs is not as useful as knowing it scmdb_people_scd1 which indicates the test case.